contracts/hooks/Vault_YieldLimitExecHook.sol

```diff
@@ -130,16 +130,24 @@ contract Vault_YieldLimitExecHook is LimitExecutionHook {
             if (pullFromStrategy) {
                 _withdrawFromStrategy(transferInfo.amount - totalIdle);
             } else {
-                (, uint256 pendingUnderlying) = abi.decode(
-                    postHookData,
-                    (uint256, uint256)
-                );
-                pendingUnderlying += totalIdle - transferInfo.amount;
+                (uint256 consumedAmount, uint256 pendingUnderlying) = abi
+                    .decode(postHookData, (uint256, uint256));
+                pendingUnderlying += transferInfo.amount - totalIdle;
```

Comment:
"amount" causes confusion since there is both underlying and shares, so we should name it consumedUnderlying and